cmd/swarm/mru.go

```diff
@@ -30,88 +30,91 @@ import (
 	"gopkg.in/urfave/cli.v1"
 )
 
-// swarm resource [--rawmru] create <name> <frequency> <0x Hexdata>
+// swarm resource create <name> <frequency> [--rawmru] <0x Hexdata>
 // swarm resource update <Manifest Address or ENS domain> <0x Hexdata>
 // swarm resource info <Manifest Address or ENS domain>
 
-func resource(ctx *cli.Context) {
-
+func resourceCreate(ctx *cli.Context) {
```

Comment:
Diff looks ugly here because this function was split in three as part of the cli refactoring.